### DISCUSSION TITLE
How can I store private user data in my hyperdrive?

### DISCUSSION THREAD

**Discussion by sonysantos** [beakerbrowser/beaker#1781] [General]
I want to create a wiki with public and private pages. However, if anyone can use the API `beaker.hyperdrive.query()`, all data is made available to all users. How can I store private pages? Is there a way to encrypt them?

**Comment by pfrazee** [comment]
Currently if you want to group public and private data into the same hyperdrive, you'll need to use encryption. We don't have any APIs for this just yet, so you'll need to implement it via your own frontends / apps

**Reply by daya** [reply]
Is there any guide one can follow to encrypt parts of the hyperdrive?
